### ISSUE TITLE
FE Skipped Headings on Billing and Insurance Top Task Template

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10062]
## Describe the defect
The current template for the content type Billing and Insurance is structured in such a way that the content does not have properly nested headings.

## To Reproduce
1. Go to [Walla Walla Billing and Insurance](https://www.va.gov/walla-walla-health-care/billing-and-insurance/)
2. View all headings on the page (you can do this using the inspector, WAVE extension, ANDI, etc.)
3. Note that headings jump from H2 > H4 skipping the H3 level heading

## AC / Expected behavior

- [ ] Phone and Hours under "Questions about copay and balance should both be H3

- [ ] Facility names should be H3

## Screenshots
![image](https://user-images.githubusercontent.com/106678594/182915579-5c093b16-7dac-4d5c-9217-4420d291ee98.png)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`
